styles.css

```diff
@@ -423,7 +430,7 @@ form {
 }
 
 .contact-button:hover {
-  background-color: #000;
+  background-color: #008ad7;
   color: white;
 }
 
```

Comment:
- [ ] I'm not able to select the specific line, so, I put this comment as close as possible. I see you're using `@media screen and (max-width: 768px)` so, you're using @media for small sizes, it would be great if @media is used for desktop size, and the default config for mobile size, in this case, you're doing invert. Please, change your style to use a mobile-first structure.